lib/sda/tree.go

```diff
@@ -258,7 +259,7 @@ func (tm TreeMarshal) MakeTree(il *EntityList) (*Tree, error) {
 
 // MakeTreeFromList creates a sub-tree given an EntityList
 func (tm *TreeMarshal) MakeTreeFromList(parent *TreeNode, il *EntityList) *TreeNode {
-	idx, ent := il.Search(tm.EntityId)
+	idx, ent := il.SearchEntity(tm.EntityId)
```

Comment:
Wait, I thought the `EntityIdx` is unique since it maps 1-to-1 to the public key? The `TreeNodeIdx` wouldn't be unique since you can run more than one `Tree` on an `Entity`.